test/test_routes.py

```diff
@@ -0,0 +1,60 @@
+"""
+.. module:: test_routes
+    :synopsis: This module tests that routes behaive as expected in Eventum.
+
+.. moduleauthor:: Dan Schlosser <dan@danrs.ch>
+"""
+
+from base import TestingTemplate
+
+
+class TestRoutes(TestingTemplate):
+    """Test the basic behavior of routes in eventum."""
+
+    OK = (200, 201)
+    REDIRECT = (301, 302)
+
+    CLIENT_ROUTE_EXPECTATIONS = [
+        ('/', OK),
+        ('/events', OK),
+        ('/events/15', REDIRECT),
+        ('/resources', OK),
+        ('/labs', OK),
+        ('/blog', OK),
+        ('/blog/15', REDIRECT),
+        ('/contact', OK),
+    ]
+
+    EVENTUM_ROUTE_EXPECTATIONS = [
```

Comment:
So right now tests are being run with google authentication off, so there isn't a great way to test that.  I'm not sure how to reconcile the need for a client_secrets file to run google auth and not wanting to publish client secrets to travis.